src/client/public/js/script.js

```diff
@@ -1,4 +1,4 @@
-class App {
+export class App {
```

Comment:
Why do we need to move this file to the `/js` subdirectory? Also, I thought that we will convert it to TS when the time comes.

Reply:
The only reason that i did that was to separate `.html` from source files. In the future, that folder may contain multiple `.js` files _(e.g. utils for `debug`)_, if we already refused idea of bundling.

Reply:
We shouldn't mix HTML files and other resources with the source files. They should be stored in a separate `/resources` or `/assets` directory. We can just add an another `app.static` for scripts.